lib/devise/token_generator.rb

```diff
@@ -19,7 +19,7 @@ def generate(klass, column)
       loop do
         raw = Devise.friendly_token
         enc = OpenSSL::HMAC.hexdigest(@digest, key, raw)
-        break [raw, enc] unless klass.to_adapter.find_first({ column => enc })
+        break [raw, enc] unless klass.exists?({ column => enc })
```

Comment:
In the another change you was using { key: value } and now { key => value }, what do you think about use one as default, I suggest { key: value }